libs/core/langchain_core/messages/utils.py

```diff
@@ -425,6 +425,8 @@ def filter_messages(
 @_runnable_support
 def merge_message_runs(
     messages: Union[Iterable[MessageLikeRepresentation], PromptValue],
+    *,
+    with_newline_separator: bool = True,
```

Comment:
thoughts on making the arg:
```chunk_separator: str="\n"```

to give end user full control

Reply:
I think so, too. I'll make the changes. Thank you for your review 😄

Reply:
@baskaryan 
I have made the necessary changes. Could you please review them?

Reply:
@baskaryan Just a friendly reminder, could you please check my changes?